### ISSUE TITLE
[Bug]: Themes by Mozilla Firefox don't have a "By Firefox" badge

### ISSUE THREAD

**Issue by leoheitmannruiz** [mozilla/addons#14947]
### What happened?

Hey :)

Themes by Mozilla Firefox don't have a "By Firefox" badge, which results in the following search not displaying results https://addons.mozilla.org/en-US/firefox/search/?promoted=line&type=statictheme

### What did you expect to happen?

I expect themes by Mozilla Firefox to have the badge "By Firefox" and that the search displays all themes by Mozilla Firefox, as it does for https://addons.mozilla.org/en-US/firefox/search/?promoted=line&type=extension

### Is there an existing issue for this?

- [X] I have searched the existing issues

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/AMOENG-967)

**Comment by diox** [comment]
Hi,

You're right, this is an inconsistency we should fix, probably by adding those themes to the right promoted group in the database.

**Comment by diox** [comment]
For reference: those themes are missing the promoted add-on state in the database (and promoted  approval I guess) are those authored by the Mozilla Firefox user, as seen here: https://addons.mozilla.org/en-US/firefox/user/4757633/ (or API https://addons.mozilla.org/api/v5/addons/search/?type=statictheme&author=4757633)

We likely need to have a migration or management command/task to fix that.

**Comment by diox** [comment]
We'll need to run a one-off command, `promote_by_firefox_themes` to fix those on every env. Any theme authored by the "Firefox" user will gain the corresponding badge once we have ran this command.

**Comment by ioanarusiczki** [comment]
I think it would be more relevant on stage since -dev has the account with 1 theme 
https://addons-dev.allizom.org/en-US/firefox/user/4757633/

Meanwhile stage https://addons.allizom.org/en-US/firefox/user/4757633/

**Comment by ioanarusiczki** [comment]
I checked this on -dev and it's looking good 

https://addons-dev.allizom.org/en-US/firefox/user/4757633/ -> all listed themes on this account have now By Firefox badge displayed, By Firefox is displayed for reviewer pages too.

**Comment by diox** [comment]
Done on stage: https://addons.allizom.org/en-US/firefox/user/4757633/

**Comment by ioanarusiczki** [comment]
Yes, it looks good. All 7 pages of themes got By Firefox badge. Making [a new version ](https://addons.allizom.org/en-US/firefox/addon/flag-of-macedonia/) with approval from rev tools would make a new By Firefox add-on version with the badge displayed on frontend.
